app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    instances = {}
+
+    for person in people:
+        person_instance = Person(person["name"], person["age"])
+        instances[person["name"]] = person_instance
+
+    for person in people:
+        if "wife" in person and person["wife"]:
```

Comment:
We can simplify it just using `if person.get("wife")` condition. Check all cases

Reply:
We can also use warlus oprator here(Inside if condition)

https://docs.python.org/3/whatsnew/3.8.html